src/stats/show_stats.py

```diff
@@ -69,11 +73,43 @@ async def show_stats(ctx, stat_id):
         make_percentage_graph(
             stat_id, options, answers, stat_title, graph_size=graph_size
         )
-        if imgur_url := await upload_graph_to_imgur_or_none(stat_id):
-            db.add_show_stats_cache(stat_id, imgur_url)
-            await send_graph_to_channel(ctx, imgur_url, stat_title)
-        else:
-            await ctx.channel.send("Server preťažený, skús prosím neskôr.")
+    elif stat_type == TIME_LINE_PLOT:
+        join_dates = []
+
+        for member in ctx.guild.members:
+            if not member.bot:
+                join_dates.append(member.joined_at)
+
+        join_dates.sort(key=lambda x: time.mktime(x.timetuple()))
+
+        server_created = ctx.guild.created_at
+        existing_for = dt.now(timezone.utc) - server_created
+
+        step = existing_for / 10
```

Comment:
Please add a comment what is this line for, what it does

Reply:
I added 2 comments that explain how the code works ([3214a02](https://github.com/StreetOfCode/discord-bot/pull/43/commits/3214a02c2989735a84a4664407f1318286ceb138)).